StringOverlayTest fails on locales with comma as decimal point

When I try to build Cantaloupe on my MacOS laptop, with the default locale set to "en_NL", the decimal point is a comma. That causes `StringOverlayTest.testToString` to fail:

```
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   StringOverlayTest.testToString:155 expected: <801774c691b35cbd89e3bd8cb6803681_SE_5_SansSerif_12_2.0_0.1_#0000FFFF_#FFA500FF_#FF0000FF_5.0_false> but was: <801774c691b35cbd89e3bd8cb6803681_SE_5_SansSerif_12_2,0_0,1_#0000FFFF_#FFA500FF_#FF0000FF_5,0_false>
```

The `String.format` call always uses the default locale for formatting, and is called in `toString`:

https://github.com/cantaloupe-project/cantaloupe/blob/fac584f55bd26f783fb4ebd02c97b82038fdcc54/src/main/java/edu/illinois/library/cantaloupe/operation/overlay/StringOverlay.java#L204-L220

Based on an [answer on StackOverflow](https://stackoverflow.com/questions/17774331/maven-default-locale-not-same-with-os-locale), the test succeeds when I change the `<argLine>` line in the Surefire plugin configuration to:
`<argLine>--add-opens java.desktop/sun.awt.image=ALL-UNNAMED -Duser.language=en_US -Duser.country=US</argLine>`.

I do wonder if the default locale needs to be set when running Cantaloupe, since overlays are dependent on them. (At least in the `toString` method, I haven't checked what that would mean in practice.)

--- a/src/main/java/edu/illinois/library/cantaloupe/operation/overlay/StringOverlay.java
+++ b/src/main/java/edu/illinois/library/cantaloupe/operation/overlay/StringOverlay.java
@@ -8,6 +8,7 @@ import edu.illinois.library.cantaloupe.util.StringUtils;
 
 import java.awt.Font;
 import java.awt.font.TextAttribute;
+import java.util.Locale;
 import java.util.Map;
 
 /**
@@ -204,7 +205,8 @@ public class StringOverlay extends Overlay implements Operation {
     public String toString() {
         // minSize is not included, as it is more of a potential property than
         // a property.
-        return String.format("%s_%s_%d_%s_%d_%.1f_%.01f_%s_%s_%s_%.1f_%s",
+        return String.format(new Locale("en", "UK"),
+                "%s_%s_%d_%s_%d_%.1f_%.01f_%s_%s_%s_%.1f_%s",
                 StringUtils.md5(getString()),
                 getPosition(),
                 getInset(),

